modules/ROOT/pages/changelog.adoc

```diff
@@ -5,4 +5,179 @@
+* Replaced the `isDisabled()` function with an `isEnabled()` function for various APIs.
+* Replaced the `enable()` and `disable()` functions with a `setEnabled(state)` function in various APIs.
+* Replaced the `disabled` property with an `enabled` property in various APIs.
+* Replaced the `disable(name)` and `enable(name)` functions with a `setEnabled(name, state)` function in the Dialog APIs.
+* Renamed the `tinymce.Env.os.isOSX` API to `tinymce.Env.os.isMacOS`.
+* Renamed the `tinymce.Env.browser.isChrome` API to `tinymce.Env.browser.isChromium` to better reflect its functionality.
+* Renamed the `getShortEndedElements` Schema API to `getVoidElements`.
+* Renamed the `font_formats` option to `font_family_formats`.
+* Renamed the `fontselect` toolbar button and `fontformats` menu item to `fontfamily`.
+* Renamed the `fontsize_formats` option to `font_size_formats`.
+* Renamed the `fontsizeselect` toolbar button and `fontsizes` menu item to `fontsize`.
+* Renamed the `formatselect` toolbar button and `blockformats` menu item to `blocks`.
+* Renamed the `styleselect` toolbar button and `formats` menu item to `styles`.
+* Renamed the `lineheight_formats` option to `line_height_formats`.
+* Renamed the `getWhiteSpaceElements()` function to `getWhitespaceElements()` in the `Schema` API.
+* Renamed the `mceInsertClipboardContent` command `content` property to `html` to better reflect what data is passed.
+* Renamed the `default_link_target` option to `link_default_target` for both `link` and `autolink` plugins.
+* Renamed the `rel_list` option to `link_rel_list` for the `link` plugin.
+* Renamed the `target_list` option to `link_target_list` for the `link` plugin.
+* The default value for the `link_default_protocol` option has been changed to `https` instead of `http`.
+* The default value for the `element_format` option has been changed to `html`.
+* The default value for the `schema` option has been changed to `html5`.
+* The default value for the `table_style_by_css` option has been changed to `true`.
+* The default value for the `table_use_colgroups` option has been changed to `true`.
+
+=== Fixed
+* The object returned from the `editor.fire()` API was incorrect if the editor had been removed.
+* The `editor.selection.getContent()` API did not respect the `no_events` argument.
+* The `editor.annotator.remove` API did not keep selection when removing the annotation.
+* The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API.
+* The `beforeinput` and `input` events would sometimes not fire as expected when deleting content.
+* The `table` plugin would sometimes not correctly handle headers in the `tfoot` section.
+* The `silver` theme UI was incorrectly rendered before plugins had initialized.
+* The aria labels for the color picker dialog were not translated.
+* Fixed sub-menu items not read by screen readers. Patch contributed by westonkd.
```

Comment:
duplication with Line 129

Reply:
This was copied from the original, so looks like we have the same issue there. I suspect this was caused by the reordering done pre 6.0 😢

Reply:
Have removed the duplicate line here but we'll also need to do the same in the tinymce repo.